application/models/GoogleSheets_Model.php

```diff
@@ -108,6 +108,60 @@ public function addNewRecord($email, $fullName, $paymentMethod)
         $result = $this->service->spreadsheets_values->update($this->spreadsheetId, $range, $body, $params);
     }
 
+    /**
+     * Mark an attendees attendance in the google sheets
+     *
+     * @param string $eventName The name of the event
+     * @param string $email The email of the user to record.
+     * @param string @upi The upi of the user if it exists
+     */
+    public function markAsPresent($eventName, $email = null, $upi = null)
+    {
+        // This should probably be in a try/catch block
+        if (is_null($email) and if_null($upi)) {
+            throw new Exception("You need to enter an email or UPI.");
+            return false;
+        }
+
+        if (is_null($eventName)) {
+            throw new Exception("You need to enter a sheet name.");
+            return false;
+        }
```

Comment:
Probably don't need this check since this is going to be an internal method we call, and `$eventName` is a required parameter.